src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -124,15 +127,15 @@ object TestUtils {
     ps.executeUpdate()
   }
 
-  def addFileMetadata(metadataId: String, fileId: String, propertyId: String): Unit = {
-    val sql = s"insert into FileMetadata (MetadataId, FileId, PropertyId, Value, Datetime, UserId) VALUES (?, ?, ?, ?, ?, ?)"
+  def addFileMetadata(metadataId: String, fileId: String, propertyName: String): Unit = {
+    val sql = s"insert into FileMetadata (MetadataId, FileId, Value, Datetime, UserId, PropertyName) VALUES (?, ?, ?, ?, ?, ?)"
```

Comment:
Upper case commands for consistency?

Reply:
I've changed this one and all of the other ones in this file.